src/main/drivers/vtx_common.h

```diff
@@ -135,6 +139,7 @@ bool vtxCommonGetBandAndChannel(const vtxDevice_t *vtxDevice, uint8_t *pBand, ui
 bool vtxCommonGetPowerIndex(const vtxDevice_t *vtxDevice, uint8_t *pIndex);
 bool vtxCommonGetFrequency(const vtxDevice_t *vtxDevice, uint16_t *pFreq);
 bool vtxCommonGetStatus(const vtxDevice_t *vtxDevice, unsigned *status);
+uint8_t vtxCommonGetPowerLevels(const vtxDevice_t *vtxDevice, uint16_t *levels, uint16_t *powers);
```

Comment:
This name might confuse developers in the future. Get which power levels? The currently set one? The ones from vtxtable?
I think it should be renamed to something more clear. Maybe `vtxCommonGetDeviceReportedPowerLevels`? Quite long though :/

Reply:
How about vtxCommonGetVTXPowerLevels()?